tests/func/test_remote.py

```diff
@@ -232,6 +232,13 @@ def test_modify_missing_remote(tmp_dir, dvc):
     assert main(["remote", "modify", "myremote", "user", "xxx"]) == 251
 
 
+def test_modify_parent_remote(tmp_dir, dvc):
```

Comment:
What's a parent remote? Just find it a bit confusing.

Reply:
Yeah, called it parent for a lack of better word. In this case it means that .dvc/config is a parent config for .dvc/config.local

Reply:
```suggestion
def test_remote_modify_local_on_repo_config(tmp_dir, dvc):
```